gasCosts/gasCosts.js

```diff
@@ -151,57 +154,90 @@ contract("All", () => {
       const STAKER2 = WORKER;
       const STAKER3 = MANAGER;
 
-      // Load the token
```

Comment:
Again something I noticed from the previous PR, the above `STAKER` assignments to constants don't read well, can we use `accounts[]` array references instead? Or maybe if they are used in more than 1 test file, move them to constants. Remove todo.

Reply:
Sure. Just tagging @lsenta to let him know your preference, given this was his handiwork originally.